bindings/gumjs/gumdukinterceptor.c

```diff
@@ -193,6 +203,8 @@ _gum_duk_interceptor_init (GumDukInterceptor * self,
   self->invocation_args = _gumjs_duk_require_heapptr (ctx, -1);
   duk_put_global_string (ctx, "InvocationArgs");
   // []
+  self->cached_invocation_args_in_use = FALSE;
```

Comment:
Let's move the _in_use initialization last, so they follow the order in the struct.